src/azurelinux/3.0/net10.0/cross/amd64-musl/Dockerfile

```diff
@@ -6,6 +6,9 @@ ARG ROOTFS_DIR
 RUN /scripts/eng/common/cross/build-rootfs.sh x64 alpine3.17 --skipunmount
 
 RUN TARGET_TRIPLE="x86_64-alpine-linux-musl" && \
+    CPP_INCLUDES=$(find "$ROOTFS_DIR/usr/include/c++/" -mindepth 1 -maxdepth 1 -type d | sort -V | head -n1) && \
+    TRIPLET_INCLUDES=$([ -e "$ROOTFS_DIR/usr/include/c++/$TARGET_TRIPLE" ] && echo "$ROOTFS_DIR/usr/include/c++/$TARGET_TRIPLE" || realpath "$ROOTFS_DIR/usr/include/../$TARGET_TRIPLE") && \
```

Comment:
The logic for this one is gnu images have it at `/usr/include/<triplet>` and musl ones have it at `/usr/include/c++/<triplet>`.

Reply:
hmm shouldn't this be `realpath "$ROOTFS_DIR/usr/include/$TARGET_TRIPLE"` then?

Reply:
copy/paste error, it's using `$CPP_INCLUDES`